src/shared/icon/index.ts

```diff
@@ -1,34 +1,62 @@
-import { html, LitElement, unsafeCSS } from "lit";
+import { html, LitElement, PropertyValues, unsafeCSS } from "lit";
 import { customElement, property } from "lit/decorators.js";
 
-import styles from "./icon.css";
+import resetCss from "../styles/reset.css";
+import iconCss from "./icon.css";
 import materialCss from "material-symbols/outlined.css?inline";
+
 import { appendFontFaceDefinitionToDom } from "../util";
 
 @customElement("bdo-icon")
 export class Icon extends LitElement {
   @property()
   icon!: string | undefined;
 
+  @property({ type: String, attribute: "icon-align" })
+  align: "start" | "center" | "end" = "end";
+  @property({ type: String, attribute: "icon-justify" })
+  justify: "start" | "center" | "end" = "end";
+  @property({ type: String, attribute: "background" })
+  background?: string;
+
+  override updated(changedProperties: PropertyValues) {
+    if (changedProperties.has("justify")) {
+      this.style.setProperty("--icon-justify", this.justify);
+    }
+    if (changedProperties.has("align")) {
+      this.style.setProperty("--icon-align", this.align);
+    }
+    if (changedProperties.has("background") && this.background) {
+      this.style.setProperty("--icon-background", this.background);
```

Comment:
Is dit nodig?

Kan de gebruiker --icon-background van een specifeke bdo-icon niet via css overschrijven?